spec/models/miq_task_spec.rb

```diff
@@ -204,6 +204,27 @@
       expect(message.zone).to eq(zone.name)
       expect(message.miq_task_id).to eq(tid)
     end
+
+    describe "#status_ok?" do
```

Comment:
I tend to avoid using another `describe` or `context` when there is only one example inside.